src/Operation/FindOne.php

```diff
@@ -54,6 +54,9 @@ class FindOne implements Executable
      *
      *  * max (document): The exclusive upper bound for a specific index.
      *
+     *  * maxAwaitTimeMS (integer): The maxium amount of time for the server to wait
+     *    on new documents to satisfy a query, if “tailable” and “awaitData” are both true.
+     *
```

Comment:
Very good practice to update the dependent operation based on changes in the parent operation class.

In this case, `maxAwaitTimeMS` doesn't serve a purpose without the `tailable` and `awaitData` options (controlled by Find's `cursorType` enum), so we don't need to mention it here.